crates/infra/cli/src/commands/test/mod.rs

```diff
@@ -42,17 +43,21 @@ impl OrderedCommand for TestCommand {
 fn test_cargo() -> Result<()> {
     CargoWorkspace::install_binary("cargo-nextest")?;
 
-    Command::new("cargo")
+    let mut command = Command::new("cargo")
```

Comment:
I think we can also reuse `add_build_rustflags()` here, to make sure binaries are reused between `cargo check` and `cargo nextest`.